assets/src/block-templates/issues/template.js

```diff
@@ -0,0 +1,115 @@
+import mainThemeUrl from '../main-theme-url';
+
+const item = ['core/group', {
+  className: 'd-flex',
+  backgroundColor: 'white',
+  style: {
+    border: { radius:'4px' },
+    spacing: {
+      padding: {
+        top:'32px',
+        right:'16px',
+        bottom:'32px',
+        left:'16px',
+      }
+    },
+  },
+  layout: {
+    type: 'flex',
+    flexWrap: 'nowrap',
+    justifyContent: "left",
+    orientation: 'horizontal',
+  },
+}, [
+  ['core/image', {
+    className: 'force-no-lightbox force-no-caption my-0 w-40',
+    url: `${mainThemeUrl}/images/placeholders/placeholder-40x40.jpg`,
+    alt: __('Enter text', 'planet4-blocks-backend'),
+    width: 40,
```

Comment:
I think that we can remove these `width` and `height` attributes, because they are overridden as soon as the image is changed (and for the placeholder image they would be set to 40 anyway since it's the correct size)